Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -183,7 +184,8 @@ class BrowserViewController: UIViewController {
             home.view.setNeedsUpdateConstraints()
         }
 
-        if let tab = tabManager.selectedTab,
+        if !isInBackground,
```

Comment:
In the commit comments I mention that I an unsure whether or not we care about this commit.  	e5886e9 fixes the actual crash, but cef4cb2 is there because when we go into the background we update the URLBar, which is kinda unecessary. e5886e9 solves the crash part. This was just belt and braces as it is only when we call `updateURLBarDisplayURL` when we go into background that I observed the crash.